addons/stock_account/tests/test_account_move.py

```diff
@@ -362,3 +362,30 @@ def test_cogs_account_branch_company(self):
         })
 
         self.assertEqual(bill.invoice_line_ids.account_id, test_account)
+
+    def test_apply_inventory_adjustment_on_multiple_quants_simultaneously(self):
+        products = self.product_a + self.product_b
+        products.write({'is_storable': True, 'categ_id': self.auto_categ.id})
+        stock_loc = self.env['stock.warehouse'].search([
+            ('company_id', '=', self.env.company.id),
+        ], limit=1).lot_stock_id
+        [self.env['stock.quant']._update_available_quantity(prod, stock_loc, qty) for prod, qty in zip(products, [5, 15])]
```

Comment:
```suggestion
        self.env['stock.quant']._update_available_quantity(self.product_a, stock_loc, 5)
        self.env['stock.quant']._update_available_quantity(self.product_b, stock_loc, 15)
```
Nitpick: Honestly, it's way harder to read compared to just having two lines :sweat_smile: 
If it would be many quants why not, but in simple cases like these readability is key.